api/db/migrations/20250930083255_add-hashedPassword-column-to-users-table.js

```diff
@@ -0,0 +1,25 @@
+const TABLE_NAME = "my_table";
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+async function up(knex) {
+  await knex.schema.createTable(TABLE_NAME, (table) => {
+    table.increments("id").primary();
+    table.string("name").notNullable();
+    table.integer("age").notNullable();
+    table.timestamp("created_at").defaultTo(knex.fn.now());
+    table.timestamp("updated_at").defaultTo(knex.fn.now());
+  });
+};
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+async function down(knex) {
+  await knex.schema.dropTable(TABLE_NAME);
+};
+
+export { down, up };
```

Comment:
This migration doesn't add a new column to users table.
```suggestion
const TABLE_NAME = "users";
const COLUMN_NAME = "hashedPassword";

/**
 * @param { import("knex").Knex } knex
 * @returns { Promise<void> }
 */
async function up(knex) {
  await knex.schema.alterTable(TABLE_NAME, (table) => {
    table.string(COLUMN_NAME).notNullable();
  });
};

/**
 * @param { import("knex").Knex } knex
 * @returns { Promise<void> }
 */
async function down(knex) {
  await knex.schema.alterTable(TABLE_NAME, () => {
    table.dropColumn(COLUMN_NAME);
  });
};

export { down, up };
```

Reply:
In the user database builder, you must add hashed password